src/main/java/fi/aalto/cs/apluscourses/model/Group.java

```diff
@@ -1,21 +1,47 @@
 package fi.aalto.cs.apluscourses.model;
 
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
+
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.jetbrains.annotations.NotNull;
 import org.json.JSONArray;
 import org.json.JSONObject;
 
 public class Group {
 
+  public static class GroupMember {
```

Comment:
Which class do you mean? `fi.aalto.cs.apluscourses.model.Student` doesn't work because it also requires a student ID, neither does `fi.aalto.cs.apluscourses.model.User` because that's something for authentication.